README.md

```diff
@@ -37,4 +37,41 @@ describe('UploadPicker rendering', () => {
 		cy.getNc(UploadPickerInput).should('exist')
 	})
 })
-```
\ No newline at end of file
+```
+
+## Starting Nextcloud Docker container
+
+It is possible to automatically start a docker container providing a Nextcloud instance for testing.
+Therefor adjust your `cypress.config.ts` (or `.js`):
+
+```js
+import { configureNextcloud,  startNextcloud,  stopNextcloud, waitOnNextcloud } from '@nextcloud/cypress/docker'
+
+export default defineConfig({
+// ...
+	e2e: {
+		// other configuration
+
+		setupNodeEvents(on, config) {
+			// Remove container after run
+			on('after:run', () => {
+				stopNextcloud()
+			})
+
+			// starting Nextcloud testing container with specified server branch
+			return startNextcloud(process.env.BRANCH)
+				.then((ip) => {
+					// Setting container's IP as base Url
+					config.baseUrl = `http://${ip}/index.php`
+					return ip
+				})
+				.then(waitOnNextcloud)
+				// configure Nextcloud, also install and enable the `viewer` app
+				.then(() => configureNextcloud(['viewer']))
```

Comment:
How can I mount the local Viewer if I'm running Cypress in https://github.com/nextcloud/viewer though?

Reply:
So, I can't use this `configureNextcloud` and have to keep mine?

Reply:
what do you mean? If you want to test the local viewer you do not need to adjust anything simply do as the readme says and the local viewer will be used

Reply:
Aaaah, if you specify it, it will not be cloned with git.
I didn't realized that. I think the jsdoc could reflect that, 
![image](https://github.com/nextcloud-libraries/nextcloud-cypress/assets/14975046/e3a92b36-32ae-4794-be71-cfd4f231675e)

Reply:
I updated the jsdocs here to reflect this.